view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -338,7 +339,7 @@ define(
                         ).done(
                         function (response) {
                             self.afterPlaceOrder();
-                            self.validateThreeDS2OrPlaceOrder(response);
+                            adyenPaymentService.getOrderPaymentStatus(response, self.validateThreeDS2OrPlaceOrder.bind(self));
```

Comment:
not sure why we need to do self.validateThreeDS2OrPlaceOrder.bind(self). I would suggest to implement a promis.